extractor/src/main/scala/com/digitalasset/extractor/config/ExtractorConfig.scala

```diff
@@ -37,3 +38,10 @@ final case class ExtractorConfig(
 object ExtractorConfig {
   type Parties = OneAnd[List, Party]
 }
+
+final case class TemplateConfig(moduleName: String, entityName: String)
```

Comment:
```suggestion
final case class TemplateConfig(moduleName: String, entityName: String) extends Ordered[TemplateConfig]
```

When possible I prefer to use `scala.math.Ordered` over `scala.math.Ordering`.

See  [`scala.math.Ordering`](https://www.scala-lang.org/api/2.12.3/scala/math/Ordering.html)
> this trait and scala.math.Ordered both provide this same functionality, but in different ways. A type T can be given a single way to order itself by extending Ordered. Using Ordering, this same type may be sorted in many other ways.